src/runtime_src/core/pcie/tools/xbutil/xbutil.cpp

```diff
@@ -1250,7 +1250,9 @@ int xcldev::device::scVersionTest(void)
     pcidev::get_dev(m_idx)->sysfs_get("xmc", "exp_bmc_ver", errmsg, exp_sc_ver);
     if (!exp_sc_ver.empty() && sc_ver.compare(exp_sc_ver) != 0)
     {
-        std::cout << "SC firmware version is not expected. SC firmware running on board: " << sc_ver << ". Expected SC firmware: " << exp_sc_ver << std::endl; 
+        std::cout << "SC firmware mismatch, ATTENTION" << std::endl;
+        std::cout << "SC firmware running on board: " << sc_ver << ". Expected SC firmware from installed Shell: " << exp_sc_ver << std::endl;
+	std::cout << "Using: xbmgmt flash --update --shell <xsabin file> to sync on board SC firmware" << std::endl;
```

Comment:
Maybe adding --card <bdf>. <bdf> could be the real one.

Reply:
this line should be a real one for user to just copy, paste and run.
will "sudo /opt/xilinx/xrt/bin/xbmgmt flash --update --card xxxx" do?

Reply:
1) I will make the change;
2) I found the other way to create this situation, which is just installing an older pkg with older SC FW version, so that even the board has newer FW, we still get this warning. I doubt that if we recommend user to just flash the card with the older pkg?
@maxzhen

Reply:
I think that we might want to cache the SC firmware version when driver is attached. Thus, it will not be impacted by what is on the disk.